minpy/core.py

```diff
@@ -125,14 +138,17 @@ def func(**kwargs):
             return executor.outputs[0]
         # Set function name to be the given symbol name.
         func.__name__ = self._sym_name
+
         # Define gradient function generator.
         def gen_grad_kw(keyname):
             def grad_wrapper(ans, **kwargs):
                 def grad_func(g):
                     executor.backward(out_grads=g)
                     ret = executor.grad_arrays[arg_names.index(keyname)]
                     return ret
+
```

Comment:
Question: are we still using per keyword gradient definition? I thought we will have a way to define one gradient for a subset of keywords (or just simply all the keywords).